src/styles/components/headers.scss

```diff
@@ -43,13 +43,12 @@ h4 {
     font-weight: 300;
     letter-spacing: 0.11rem;
     line-height: 3rem;
-    padding: 4rem 8rem;
+    padding: 2rem 2rem 11rem;
```

Comment:
Do hero's need this much padding on the bottom and so little on top? Also, I think the padding for the gutters on each side are 1.5rem (at least this is the value ive been using everywhere)